### DISCUSSION TITLE
Microsoft Visual c++ runtime error

### DISCUSSION THREAD

**Discussion by srsneha23** [carla-simulator/carla#5773] [Q&A]
whenever i open carla it gives microsoft visual c++ runtime error, I was able to clear the directx runtime error but not the visual c++ 
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/114008757/191301622-2fafee7e-4658-4f5a-bcfd-4e53137415cf.png)

**Comment by ishaan95** [comment]
Are you running Carla using the x64 command prompt? This is explained in the link below. Also, have you installed all Visual Studio packages as mentioned here - https://carla.readthedocs.io/en/0.9.13/build_windows/#visual-studio-2019?
